expressjs/db/migrations/20200526095904_rebuild.js

```diff
@@ -0,0 +1,112 @@
+const { onUpdateTrigger } = require('../../knexfile');
+
+function dropTables(knex) {
+  let dropQuery = `
+    DROP TABLE if exists publications;
+    DROP TABLE if exists trails;
+    DROP TABLE if exists users;
+    DROP TABLE if exists cases;
+    DROP TYPE if exists state_type;
+    DROP TABLE if exists organizations;
+  `;
+  return knex.raw(dropQuery);
+}
+
+function buildOrganization(knex) {
+  return knex.schema.createTable('organizations', function (table) {
+    table.uuid('id').notNull().primary();
+    table.string('authority_name')
```

Comment:
This should potentially be just `name`. A health authority is comprised of many organizations so the use `authority` here should be a little misleading.